components/layout/PageHeader.tsx

```diff
@@ -14,7 +15,7 @@ function PageHeader() {
 
       <ErrorBoundary>
         <NextLink href="/" passHref>
-          <Link>
+          <Link className="js-headerLogo">
```

Comment:
something to hook onto in tests.. should probably be `test-e2e-headerLogo` or `test-headerLogo`